main.py

```diff
@@ -16,7 +16,7 @@
 
 # Enable logging
 logging.basicConfig(
-    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO
+    format="%(asctime)s - %(levelname)s - %(message)s", level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S'
```

Comment:
Tweaked this to reduce detail in error message eg miliseconds and function name.

Reply:
Probably OK for a small app like this 👍